slackbot/dispatcher.py

```diff
@@ -177,3 +185,7 @@ def channel(self):
     @property
     def body(self):
         return self._body
+
+    def docs_reply(self):
+        reply = ['    • `{0}` {1}'.format(v.__name__, v.__doc__ or "") for p, v in iteritems(self._plugins.commands['respond_to'])]
```

Comment:
> well, I did to be able to display the docs of the plugins whenever you want as of here

Now I see that. The `PluginsManager` keeps the list of commands in a class variable, so no need to initialize it again here.

Still, it's not good to duplicate the logic twice (both here and in the `MessageDispatcher._default_rely()` method):
```python
['    • `{0}` {1}'.format(v.__name__, v.__doc__ or "") for p, v in iteritems(self._plugins.commands['respond_to'])]
```

We should create a new method in the `PluginsManager` class to wrap it, and call the new method  in the two places.